src/utils/types.ts

```diff
@@ -25,20 +25,24 @@ export interface KmsService {
 }
 
 export interface ActivityLogEntry {
-  event_type: string;
-  session_id: string;
   user_id: string;
   timestamp: number;
-  activities?: Activity[];
-  truncated: boolean;
+  client_id: string;
+  event_id: string;
+  event_type: string;
+  reported_suspicious: boolean;
+  session_id: string;
 }
 
-export interface Activity {
-  type: string;
-  client_id: ClientId;
-  timestamp: number;
+export interface FormattedData {
```

Comment:
This name isn't very informative.  Could it be changed to FormattedActivityLog?  Was this copied from the report suspicious activity branch?

Reply:
Done